src/FDC3NotificationService/index.tsx

```diff
@@ -0,0 +1,255 @@
+import React from "react";
+// This line imports type declarations for Finsemble's globals such as FSBL and fdc3. You can ignore any warnings that it is defined but never used.
+// Please use global FSBL and fdc3 objects instead of importing from finsemble-core.
+import {
+	AppIdentifier, Channel,
+	ChannelError,
+	Context,
+	IntentHandler,
+	StandardErrorCallback,
+	types
+} from "@finsemble/finsemble-core";
+import {
+	IAction,
+	IFilter,
+	INotification,
+	ISubscription
+} from "@finsemble/finsemble-core/types/platform/services/notification/types";
+import {RouterResponse} from "@finsemble/finsemble-core/types/platform/services/router/types";
```

Comment:
Should these types be exported as part of `@finsemble/finsemble-core`?

Reply:
I think so...

I will create a ticket to export them